docs/API.md

```diff
@@ -1029,6 +1029,33 @@ minioClient.deleteBucketNotification('my-bucketname', function(e) {
 })
 ```
 
+<a name="getBucketPolicy"></a>
+### getBucketPolicy(bucketName, objectPrefix, callback)
+
+Get the bucket policy associated with the specified bucket. If `objectPrefix`
+is not empty, the bucket policy will be filtered based on object permissions
+as well.
+
+__Parameters__
+
+
+| Param  |  Type | Description  |
+|---|---|---|
+| `bucketName`  | _string_  | Name of the bucket |
+| `objectPrefix` | _string_ | Prefix of objects in the bucket with which to filter permissions off of. Use `''` for entire bucket. |
+| `callback(err, policy)`  | _function_  | Callback function is called with non `null` err value in case of error. `policy` will be the string representation of the bucket policy (`none`, `readonly`, `writeonly`, or `readwrite`). |
```

Comment:
if JS was a typed language i could have argued that using BUCKET_POLICY_NONE is better than 'none' string because if the developer made a mistake in typing say BUCKET_POLICY_NONEEE he would get a compile time error and not run time error, but if he wrote 'noneee' compiler would not be able to detect that error.

But in JS the developer will not get  a compile error even if he types minio.BUCKET_POLICY_NONEEE so there is no advantage for JS.
But it's better to be consistent with other minio libraries which use constants for policy names, so you can use:
Policy.NONE
Policy.READONLY
Policy.READWRITE
Policy.WRITEONLY

and

```
export const Policy = {
    NONE: 'none',
    READWRITE: 'readwrite',
    ...
}
```

you'll need to reexport it in minio.js like how it's done for notification.js